include/types.hpp

```diff
@@ -14,6 +14,13 @@ namespace vpd
 {
 namespace types
 {
+
+using BiosProperty = std::tuple<
```

Comment:
can we move this after BinaryVector , as basic type declaration should stay top. ?

Please ignore if this is not the case.

Reply:
Ignored :D